activesupport/test/core_ext/range_ext_test.rb

```diff
@@ -60,6 +60,16 @@ def test_should_include_other_with_exclusive_end
     assert((1..10).include?(1...11))
   end
 
+  if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.6.0")
+    def test_include_with_endless_range
+      assert(eval("1..").include?(2))
+    end
+
+    def test_include_with_beginless_range
```

Comment:
Beginless ranges are an upcoming feature in Ruby 2.7; this test will needs its own version guard.